packages/graphiql-explorer/README.md

```diff
@@ -0,0 +1,156 @@
+Interactive explorer plugin for GraphiQL.
+
+Try it live with the [GraphiQL Plugins Demo](https://graphiql-test.netlify.app/webpack).
+
+## Created by OneGraph
+
+[OneGraph](https://www.onegraph.com) provides easy, consistent access to the APIs that underlie your business--all through the power of GraphQL.
+
+Sign up at [https://www.onegraph.com](http://www.onegraph.com).
+
+[![npm version](http://img.shields.io/npm/v/graphiql-explorer.svg?style=flat)](https://npmjs.org/package/graphiql-explorer 'View this project on npm')
+
+## Example usage
+
+The recommended way to use `graphiql-explorer` is via the `graphiql` plugin - [`@graphiql/plugin-explorer`](../graphiql-plugin-explorer/)
+
+[Read the rationale on the OneGraph blog](https://www.onegraph.com/blog/2019/01/24/How_OneGraph_onboards_users_new_to_GraphQL.html).
+
+## Changes from the OneGraph version
+
+1. Converted to Typescript
+2. Split into multiple files
+3. new `ExplorerInner` and `GraphiQLExplorerInnerProps` exports which remove some of the presentation complexity for the graphiql plugin.
+4. uses modern targets and only exports esm build, does not export cjs, bundle and transpile as needed
+
+## Customizing styles
+
+The default styling matches for the Explorer matches the default styling for GraphiQL. If you've customized your GraphiQL styling, you can customize the Explorer's styling to match.
+
+### Customizing colors
+
+The Explorer accepts a `colors` prop as a map of the class names in GraphiQL's css to hex colors. If you've edited the GraphiQL class names that control colors (e.g. `cm-def`, `cm-variable`, `cm-string`, etc.) use those same colors in the colors map. The naming of the keys in the colors map tries to align closely with the names of the class names in GraphiQL's css (note that the Explorer can't just apply the classes because of conflicts with how the css file styles inputs).
```

Comment:
```suggestion
The Explorer accepts a `colors` prop as a map of the class names in GraphiQL's css to hex colors. If you've edited the GraphiQL class names that control colors (e.g. `cm-def`, `cm-variable`, `cm-string`, etc.), use those same colors in the colors map. The naming of the keys in the colors map tries to align closely with the class names in GraphiQL's css (note that the Explorer can't just apply the classes because of conflicts with how the css file styles inputs).
```